release_tools/changelog.py

```diff
@@ -211,13 +212,33 @@ def create_changelog_entry_content(title, category, author=None, issue=None,
     return content
 
 
-def write_changelog_entry(dirpath, title, content, overwrite=False):
-    """Store the contents of an entry in a file."""
-
+def validate_changelog_content(content, run_editor=True):
     if not content:
         msg = "Aborting due to empty entry content"
         raise click.ClickException(msg)
 
+    while True:
+        try:
+            data = yaml.safe_load(content)
+            return content
+        except yaml.YAMLError as exc:
+            pm = exc.problem_mark
+            click.echo("Error: File has an issue on line {} at position {}".format(pm.line, pm.column))
+
+            if not click.confirm("Do you want to edit it?"):
+                msg = "Aborting due to wrong file format"
```

Comment:
Should this abort or just save the entry with the errors?

Reply:
If the user selected "N" (no), it would abort, doesn't save the file with errors.

Reply:
If the user selects "y" (yes), it would show the errors and open the editor. This process would continue either the user selects no or corrects all the errors.

Reply:
> Should this abort or just save the entry with the errors?

saving the entry might create erros later, so I implemented abort :thinking: 
WDYT?